### PR TITLE
Update the 'Team' page to reflect recent promotions. #1235

### LINKED ISSUE #1235: Update the 'Team' page to reflect recent promotions.
Area leads:
Dat  - API, Performance, 
Joshua - UI, Config, Testing,
Thien - Storage, Filters, Threading

Senior developers:
Hans,
Manmeet 

Committers:
Xinan, Martin, Yi Yan, Soon Jin, Rais

### PR DESCRIPTION
**Pull request by YijinL**
Fixes #1235.

### CODE REVIEW

**Comment by dariusf** [comment]
I think this looks fine. Could you squash everything? Since it's conceptually one change.

**Comment by dariusf** [comment]
Oh, Rajan should probably be a past contributor, since he has graduated. Not sure.

File: docs/team.md [hunk 1]
```diff
@@ -1,26 +1,54 @@
...
+- [Joshua Lee](http://github.com/lejolly)     {**UI**|**Config**|**Testing**}         (Team member since May 2015)
+- [Nguyen Quoc Dat](http://github.com/acruis) {**API**|**Performance**}               (Team member since May 2015)
+- [Thien Nguyen] (https://github.com/ndt93)   {**Storage**|**Filters**|**Threading**} (Team member since July 2015)
+
+###### Senior Developers
+
+- [Hans Adrian](http://github.com/HansNewbie) (Team member since Aug 2015)
+- [Manmeet Saluja](http://github.com/codemanmeet) (Team member since Aug 2015)
+
+###### Members
```
Comment (damithc) [inline docs/team.md]: Change heading to `Committers`

File: docs/team.md [hunk 2]
```diff
@@ -1,26 +1,54 @@
...
+
+- [Rajan Dhir](http://github.com/rd1992) (Team member since Jan 2015)
```
Comment (damithc) [inline docs/team.md]: Rajan should be moved to `past team members`

File: docs/team.md [hunk 3]
```diff
@@ -1,26 +1,54 @@
...
 
 - [Hyung Jon] (https://github.com/HyungJon)
 - [Ian Ngiaw] (https://github.com/ianngiaw)
-- [Su Gi Chandran] (https://github.com/Honoo)
\ No newline at end of file
+- [Jiang Sheng](https://github.com/Gisonrg)
+- [Liu Xinan](https://github.com/xinan)
+- [Martin Choo Boon Yong](https://github.com/m133225)
+- [Muhammad Rais Fathin Mudzakir](https://github.com/raisfathin)
+- [Nicholas Lum](https://github.com/nicholaslum444)
```
Comment (damithc) [inline docs/team.md]: This 4, Soon Jin, and Martin should go under committers.

File: docs/team.md [hunk 4]
```diff
@@ -1,26 +1,54 @@
...
+- [Raviprakash Chandak](https://github.com/ravic1995)
+- [Soon Jin](https://github.com/jinified)
+- [Su Gi Chandran] (https://github.com/Honoo)
+- [Wang Jie](https://github.com/gaieepo)
+- [Yi Yan](https://github.com/garbanzos)
+- [Zheng Weihan](https://github.com/whattokingu)
+- [Zhongwei Zhang](https://github.com/zhongwei-z)
```
Comment (damithc) [inline docs/team.md]: you can add yourself here.

File: docs/team.md [hunk 5]
```diff
@@ -1,26 +1,54 @@
...
-- [Tan Wei Lin](http://github.com/xeluna) (team member: Jul 2014 - Dec 2014)
-- [Wong Jun Wei](http://github.com/jwwong) (team member: Jul 2014 - Dec 2014)
-- [Naomi Leow](http://github.com/naomilwx) (team member: Jul 2014 - Aug 2014)
+- [Eric Wang Gaoxiang](http://github.com/wgx731)  (team member: April 2015 - May 2015)
+- [Naomi Leow](http://github.com/naomilwx)        (team member: Jul 2014 - Aug 2014)
+- [Tay Chun Mei](http://github.com/taychunmei)    (team member: Jul 2014 - Dec 2014)
+- [Tan Wei Lin](http://github.com/xeluna)         (team member: Jul 2014 - Dec 2014)
+- [Wong Jun Wei](http://github.com/jwwong)        (team member: Jul 2014 - Dec 2014)
 
 ## Current Contributors
```
Comment (damithc) [inline docs/team.md]: Remove `Current`

**Comment by damithc** [comment]
Some changes suggested.

File: docs/team.md [hunk 6]
```diff
@@ -1,26 +1,54 @@
...
+- [Hans Adrian](http://github.com/HansNewbie) (Team member since Aug 2015)
+- [Manmeet Saluja](http://github.com/codemanmeet) (Team member since Aug 2015)
+
+###### Committers
+
+- [Liu Xinan](https://github.com/xinan)
+- [Martin Choo Boon Yong](https://github.com/m133225)
+- [Muhammad Rais Fathin Mudzakir](https://github.com/raisfathin)
+- [Nicholas Lum](https://github.com/nicholaslum444)
+- [Soon Jin](https://github.com/jinified)
```
Comment (damithc) [inline docs/team.md]: Yi Yan should be in this list.

**Comment by YijinL** [comment]
squashed and updated Yi Yan (contributor -> committer).

**Comment by damithc** [comment]
@acruis can merge this one?

**Comment by acruis** [comment]
Alright.

**Pull request merged**